tests/core/pyspec/eth2spec/test/merge/unit/test_unit.py

```diff
@@ -0,0 +1,122 @@
+from eth2spec.utils.ssz.ssz_typing import uint256
```

Comment:
The name of this file is not descriptive enough.

I suggest `test/merge/unittests/test_transition_unittest.py`

Reply:
or shorter, `test/merge/unittests/test_transition.py`